cmd/goal/ledger.go

```diff
@@ -17,13 +17,25 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
-
+	msgpackcore "github.com/algorand/go-algorand/cmd/msgpacktool/core"
```

Comment:
Not commenting on the merit of this change, but:
1) imports should be sorted properly
2) We do not import from source under ./cmd.  If you want to reuse functionality, it should be broken out into a separate package NOT under ./cmd.